packages/govuk-frontend/src/govuk/components/tabs/tabs.mjs

```diff
@@ -51,11 +51,7 @@ Tabs.prototype.init = function () {
     return
   }
 
-  if (typeof window.matchMedia === 'function') {
-    this.setupResponsiveChecks()
-  } else {
-    this.setup()
-  }
+  this.setupResponsiveChecks()
```

Comment:
Another non-blocker, but shall we delete `setupResponsiveChecks()` and move its code into `init()`?

There's only one "init path" through the code now, like the other components

Reply:
I decided not to do that because we're saying these methods will go private in v5, so it felt like moving 'em outright is either a "later" thing, or one for discussion when we review the code of the relevant components.